lib/simple_form/components/labels.rb

```diff
@@ -50,7 +53,7 @@ def label_html_options
           label_options[:for] = options[:input_html][:id]
         end
 
-        label_options
+        label_options.except(*VIRTUAL_OPTIONS)
```

Comment:
Oh, we should be filtering it here https://github.com/plataformatec/simple_form/blob/81b9d7d3439225c7613e570afb44596acfa97a76/lib/simple_form/form_builder.rb#L302